lib/action_tracker/base.rb

```diff
@@ -1,19 +1,20 @@
 module ActionTracker
   # nodoc
   class Base
-    attr_reader :kontroller, :resource
+    attr_reader :controller, :resource
+    alias_method :kontroller, :controller
```

Comment:
We're doing a minor patch upgrade now, and is important that we maintain the compatibility within minor upgrades. Before change the public API of a class we need to WARNING in some pre-release versions and then we're good to deprecate.